crypto/src/transaction/builder.rs

```diff
@@ -119,7 +117,21 @@ impl Builder {
             ovk_wrapped_key,
         });
 
-        self
+        (note, self)
+    }
+
+    /// Create a new `Output`, implicitly creating a new note for it and encrypting the provided
+    /// [`MemoPlaintext`] with a fresh ephemeral secret key.
```

Comment:
similarly to the above, maybe the doc comments should be unified, with this one noting that it is chainable?